src/common/RunParams.cpp

```diff
@@ -1140,9 +1142,13 @@ void RunParams::parseCommandLineOptions(int argc, char** argv)
         input_state = DryRun;
       }
 
-    } else if ( std::string(argv[i]) == std::string("--disable-warmup") ) {
+    } else if ( std::string(argv[i]) == std::string("--warmup-disable") ) {
+
+      warmup_mode = WarmupMode::Disable;
 
-      disable_warmup = true;
+    } else if ( std::string(argv[i]) == std::string("--warmup-to-run") ) {
```

Comment:
I'm not quite sure that this name is clear. I read it as "warmup to run ... what?"